presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/GeometryBenchmarkUtils.java

```diff
@@ -34,4 +34,20 @@ public static String loadPolygon(String path)
         String[] parts = line.split("\\|");
         return parts[0];
     }
+
+    public static String createPolygon(double longitude, double nbVertices)
+    {
+        StringBuilder sb = new StringBuilder();
+        sb.append("POLYGON((");
+        String separator = "";
+        for (int i = 0; i < nbVertices; i++) {
+            sb.append(separator);
+            sb.append(longitude);
```

Comment:
this is not a valid polygon, is it? I believe it looks like a line (has no area). Is this intentional?

Reply:
I see it as a slice of the northern hemisphere

Reply:
which I got wrong, will fix !